evaluator/evaluate_packages.go

```diff
@@ -63,36 +63,63 @@ func lazySavePackages(tx *gorm.DB, names, evras []string) error {
 	return nil
 }
 
+type packageMetadata struct {
+	NameID          int
+	Name            string
```

Comment:
Is the name necessary here? It's a key in name2PackageMetadataCache map so you already know it when returning this struct from map. And if I see correctly it isn't used later anyway.